model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/organization/CachedOrganization.java

```diff
@@ -55,10 +55,10 @@ public CachedOrganization(Long revision, RealmModel realm, OrganizationModel org
 
     @Override
     public String getRealm() {
-        return realm.getId();
+        return realm;
     }
 
-    public RealmModel getRealmModel() {
+    public String getRealmModel() {
```

Comment:
can it be removed? Invokers can use `getRealm()` method above.

Reply:
+1

Reply:
Done